### DISCUSSION TITLE
HyperHDR iphone shortcut help

### DISCUSSION THREAD

**Discussion by chesterflaps** [awawa-dev/HyperHDR#82] [Q&A]
Hi all.

I am very new to the Raspberry pi, hyperion and ssh arena so forgive me if I sound clueless (it's because I am).

I want to be able to turn the LED's off without having to open my laptop so I downloaded the Hyperion app. There does not appear to be a button to turn on and off like there is on the webUI. All I can see id a slider which I can slide all the way down and they go off but not sure if this is doing the same thing. The lights are going off but I would have expected an on/off button.

Secondly, I am trying to create an iPhone shortcut to do the same thing so I can just start and stop hyperion with one press on my iphone. I have found this code online '

Code
sudo systemctl start hyperion.service
and

Code
sudo systemctl stop hyperion.service

When adding in the iphone shortcuts, it asks for 
Host - My Pi IP address I assume
Port - the same as the one showing on HyperHD WebUI I assume
User - not sure
Authentication - either Password or SSH key. No idea what this is
Password - Is this my Pi password?

I have tried a few different ways and the shortcut dimply does not work. Can anyone offer any advice?

I just want to be able to turn the lights off easily. Alexa control would be best but I don't think that is possible.

Any help would be greatly appreciated.

**Comment by awawa-dev** [comment]
Hi
First of all:
- Hyperion app is outdated, even for the hyperion ng. Hyperhdr doesn't support it. All can be done using web browser (and the next version will be even much more friendly for mobile browser) and I don't see any point in using mobile apps anyway.
- there is no service hyperion.service for hyperhdr (check docs: https://hyperhdr.blogspot.com/2020/11/hyperhdr-prepare-for-building-buying_17.html#chapter3 example for pi user)
- don't know exactly what tool you are configuring, seems like remote ssh command so its rather ssh default port, authentication is rather 'password' unless you changed it on the system somehow, default password for Raspbian/Raspberry Pi OS you can find also in the link above
- you can turn off/on leds using remote API (with commands passed for example using JSON helper https://github.com/awawa-dev/HyperHDR/releases/tag/v16.0.0.1) but hyperhdr service must be running all the time and it's rather advanced thing

**Comment by Ar-Cero** [comment]
Hey u guys, 

first of all, thanks a lot for HyperHDR, I was using HyperCon and Hyperion on Raspbian, but honestly, hyperhdy is what I actually was looking for. Movies without HDR Ambientlight are simply not the same.

Nevertheless, the Hyperion mobile app was from my point of view also a little handy, because of the simple few most basic options, e.g. ambient lighting without any movie or something, only to have some light in the room. So Chesterflaps, here is a little simple compromise I did: Connet to Hyper HDR via your browser - add to favorites - add a shortcut to the favorites on your mobile phone screen. - u ll find switch on of on the first page you see, for somme blue mood bubbles - go to remote control - scroll down to colors/effects - and choose your effect ;)

thanks again for hyperhdr!!

regards from Minna
